recipes/pykicad/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "pykicad" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ded058f3af116845d8a0495f3a8ff4b6a7e707e534c5b71ee61b37fb23553923
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - pyparsing
+
+test:
+  imports:
+    - pykicad
+
+about:
+  home: https://github.com/dvc94ch/pykicad
+  license: ISC
+  # ISC throws an error if used as the family. I think BSD is appropriate
+  license_family: BSD
```

Comment:
the license family should be OTHER

Reply:
Really?

>  It is functionally equivalent to the simplified BSD and MIT licenses, but without language deemed unnecessary following the Berne Convention.[nb 1][nb 2]

https://en.wikipedia.org/wiki/ISC_license

Reply:
MIT and BSD are not functionally equivalent, so I don't trust this.  It isn't a BSD license